pkgs/development/python-modules/genson/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "genson";
+  version = "1.2.2";
+  format = "setuptools";
```

Comment:
This package uses `pyproject.toml` and setuptools as build backend.

https://github.com/wolverdude/GenSON/blob/2121a2450ceff5d90c6cabe4be8d6553c2ebca6d/pyproject.toml#L1-L3

Reply:
Good catch, fixed that